app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Api
+  # Register users
+  class UsersController < ApplicationController
+    def create
+      user = User.new(user_params)
+
+      if user.save
+        # TODO: replace with real location
+        render json: user, status: :created, location: nil
+      else
+        render json: user, status: :unprocessable_entity
+      end
+    end
+
+    private
+
+    def user_params
+      params.require(:user).permit(:email, :name, :password)
+    end
```

Comment:
Are you going to use interactors?

Reply:
No :)

Reply:
Why 👎 ? I don't think that interactors is a good thing. Anyway it is my pet project so I will try to explore new verges and use OOP instead.

Reply:
IMHO interactors are the best way to separate business and application logic from other service stuff but I agree that in pet project you are able to try other ways